docs/comlink/reference/binary_data.mdx

```diff
@@ -0,0 +1,198 @@
+---
+slug: /comlink/reference/map/binary_data
+displayed_sidebar: referenceSidebar
+---
+
+# Binary Data
+
+:::info
+
+To use this feature, you will need OneSDK for NodeJS in version `2.2.0` or higher.
+
+:::
+
+To use binary data to upload content OneSDK has `BinaryData` class.
```

Comment:
Maybe something like "Use `BinaryData` class in OneSDK to upload content by streaming bytes. The following interface is available in Comlink Maps:"

I think this leading sentence is the most important part of this doc as it will tell the user what to expect from `BinaryData`, so we should definitely mention streaming. I think we should also mention that is it both a class in Javascript SDK as well as an interface in Comlink script.